packages/adapter-postgres/package.json

```diff
@@ -36,7 +36,8 @@
   },
   "dependencies": {
     "pg": "^7.7.1",
-    "squel": "^5.12.2"
+    "squel": "^5.12.2",
+    "@fewer/sq": "^0.1.0"
```

Comment:
is this a required dep or just a dev dep?

Reply:
Probably just devDep